docs/reference/modules/discovery/bootstrapping.asciidoc

```diff
@@ -7,19 +7,13 @@ more of the master-eligible nodes in the cluster. This is known as _cluster
 bootstrapping_.  This is only required the very first time the cluster starts
 up: nodes that have already joined a cluster store this information in their
 data folder and freshly-started nodes that are joining an existing cluster
-obtain this information from the cluster's elected master. This information is
-given using this setting:
+obtain this information from the cluster's elected master. 
 
-`cluster.initial_master_nodes`::
-
-    Sets a list of the <<node.name,node names>> or transport addresses of the
-    initial set of master-eligible nodes in a brand-new cluster. By default
-    this list is empty, meaning that this node expects to join a cluster that
-    has already been bootstrapped.
-
-This setting can be given on the command line or in the `elasticsearch.yml`
-configuration file when starting up a master-eligible node. Once the cluster
-has formed this setting is no longer required and is ignored. It need not be set
+The initial set of master-eligible nodes is defined in the 
+<<modules-discovery-settings,`cluster.initial_master_nodes` setting>>. When you
```

Comment:
I think it would be friendlier to link to the "important settings" page here, i.e. `<<initial_master_nodes>>`.